test/karpenter.test.ts

```diff
@@ -428,4 +428,28 @@ describe('Unit tests for Karpenter addon', () => {
         expect(manifest.spec.blockDeviceMappings.length).toEqual(1);
         expect(manifest.spec.blockDeviceMappings[0]).toMatchObject(blockDeviceMapping);
       });
+
+      test("Stack creation succeeded using KubernetesVersion.Of", () => {
+        const app = new cdk.App();
+
+        const blueprint = blueprints.EksBlueprint.builder();
+
+        blueprint.account("123567891").region('us-west-1')
+            .version(KubernetesVersion.of('1.28'))
```

Comment:
Why are we doing `.of`? Can we not use `eks.KubernetesVersion.V1_28` ?

Reply:
It should be able possible to use both style... This PR allows both ways to set the version.